### DISCUSSION TITLE
posts/designing-a-blog-microfeature-for-better-immersion

### DISCUSSION THREAD

**Discussion by giscus** [poodlepoodle/poodle-blog-comments-giscus#3] [Comments]
# posts/designing-a-blog-microfeature-for-better-immersion

새로운 기술이 파도처럼 몰려와도 지워지지 않을 개발자국을 남깁니다.

https://www.poodlepoodle.me/posts/designing-a-blog-microfeature-for-better-immersion

**Comment by giljihun** [comment]
우와... 블로그 너무 이뻐요 ㅋㅋㅋ 글 잘보고 갑니다 ㅎㅎ

**Reply by poodlepoodle** [reply]
지훈님!!! 감사드려요 ㅎㅎㅎㅎ
